mypy-baseline.txt

```diff
@@ -892,13 +893,8 @@ src/inmanta/execute/scheduler.py:0: error: Missing type parameters for generic t
 src/inmanta/execute/scheduler.py:0: error: "<subclass of "DelayedResultVariable" and "RelationAttributeVariable">" has no attribute "attribute"  [attr-defined]
 src/inmanta/execute/scheduler.py:0: error: "<subclass of "DelayedResultVariable" and "RelationAttributeVariable">" has no attribute "attribute"  [attr-defined]
 src/inmanta/execute/scheduler.py:0: error: "<subclass of "DelayedResultVariable" and "RelationAttributeVariable">" has no attribute "attribute"  [attr-defined]
+src/inmanta/env.py:0: error: Argument 1 to "parse_requirements" has incompatible type "Sequence[Requirement]"; expected "Sequence[str]"  [arg-type]
```

Comment:
this looks suspicious?

Reply:
I don't know why the line is not reported correctly but the actual error is:
```
src/inmanta/env.py:176: error: Argument 1 to "parse_requirements" has incompatible type "Sequence[Requirement]"; expected "Sequence[str]"  [arg-type]
```

This is the code triggering this error:
```
    def _get_as_requirements_type(cls, requirements: req_list) -> Sequence[packaging.requirements.Requirement]:
        """
        Convert requirements from Union[Sequence[str], Sequence[Requirement]] to Sequence[Requirement]
        """
        if isinstance(requirements[0], str):
            return util.parse_requirements(requirements)
        else:
            return requirements
```